network/wifi/at_parser.c

```diff
@@ -317,7 +303,13 @@ static inline int32_t check_conn_id(struct at_desc *desc, struct at_buff *msg,
 		}
 	}
 	if (id >= 0 && id < 4) {//Maximum of 4 connections
+		/* Close connection */
+		desc->current_conn = id;
 		desc->conn[id].active = false;
+		desc->conn[id].cbuff = NULL;
+		/* Notify that a connection was closed */
+		desc->connection_callback(desc->callback_ctx,
+					  AT_CLOSED_CONNECTION, id, NULL);
```

Comment:
release the circular buffer asociated with the connection

Reply:
it is released in wifi.c